test/chemistry/test_swaprz.py

```diff
@@ -42,8 +45,10 @@ def test_excitation_preserving(self):
 
         driver = HDF5Driver(self.get_resource_path('test_driver_hdf5.hdf5'))
         qmolecule = driver.run()
+        warnings.filterwarnings('ignore', category=DeprecationWarning)
```

Comment:
Please just switch out the component that are used to test things here - we should not have to duplicate tests and run twice as many. Though arguably in this case the goal was to test SwapRZ var form which no longer exists and is a circuit in library so it should be tested there - but I get why we want to check it continues to work as expected with chemistry.